scripts/documentation_provider.py

```diff
@@ -205,7 +205,8 @@ def print_entry(
 
     def print_events(self, class_name: str) -> None:
         clazz = self.classes[class_name]
-        if events := clazz["events"]:
+        events = clazz["events"]
+        if events:
```

Comment:
Why can we no longer use `:=` here? This code only runs in 3.8+. I cannot reproduce it locally, and in CI mypy complains, but this seems like a FP and I cannot reproduce locally.

It's unideal the the pre-commit run locally vs. CI produces different results. Please file an issue to sort that out as this change seems to be hiding a larger issue with our linting.

Reply:
We chatted offline. We're still not sure—and while not satisfying—since this seems to be a one off, this is good workaround for the lint FP for now. 

If more discrepancies arise, we can spend more time digging in!